oak_runtime/src/node/http/server.rs

```diff
@@ -56,6 +56,9 @@ use oak_abi::proto::oak::identity::SignedChallenge;
 use prost::Message;
 use tokio_rustls::TlsAcceptor;
 
+// Workaround for https://rust-lang.github.io/rust-clippy/master/index.html#borrow_interior_mutable_const.
+static TRANSFER_ENCODING: http::header::HeaderName = http::header::TRANSFER_ENCODING;
```

Comment:
Is it needed to define a constant in the code here that points directly to a constant in another library?

Reply:
Yes. Because of clippy's `borrow_interior_mutable_const` error. I could not find a better solution.

Reply:
@conradgrobler note that strictly speaking this is just a static binding, not a const. It is "instantiating" the const value once, which otherwise gets inlined in every call place (hence the warning, since instances of const items are always distinct).